frontend/appflowy_flutter/lib/plugins/database_view/widgets/row/row_detail.dart

```diff
@@ -43,147 +45,100 @@ class RowDetailPage extends StatefulWidget with FlowyOverlayDelegate {
 }
 
 class _RowDetailPageState extends State<RowDetailPage> {
-  final padding = const EdgeInsets.symmetric(
-    horizontal: 40,
-    vertical: 20,
-  );
-
   @override
   Widget build(BuildContext context) {
     return FlowyDialog(
       child: BlocProvider(
         create: (context) {
-          final bloc = RowDetailBloc(
-            dataController: widget.dataController,
-          );
-          bloc.add(const RowDetailEvent.initial());
-          return bloc;
+          return RowDetailBloc(dataController: widget.dataController)
+            ..add(const RowDetailEvent.initial());
         },
-        child: Padding(
-          padding: padding,
-          child: Column(
-            children: [
-              const _Header(),
-              Expanded(
-                child: _PropertyColumn(
-                  cellBuilder: widget.cellBuilder,
-                  viewId: widget.dataController.viewId,
-                ),
+        child: ListView(
+          children: [
+            // const SizedBox(height: 100),
+            // const Divider(height: 1.0),
+            IntrinsicHeight(
+              child: Row(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                mainAxisSize: MainAxisSize.min,
+                children: [
+                  Flexible(
+                    flex: 4,
```

Comment:
Are we sure that the flex factor should always be `4`? How does the property column look when the app is on a small screen? An idea here would be to use a greater flex factor when the screen width is smaller. 

```dart
enum FormFactor {
  small._(0),
  medium._(800),
  large._(1200);
  
  const FormFactor._(this.width);
  
  factory FormFactor.from(Size size) {
    final width = size.width;
    if (width > FormFactor.large.width) {
      return FormFactor.large;
    }
    
    if (width > FormFactor.medium.width) {
      return FormFactor.medium;
    }
    
    return FormFactor.small;
  }
  
  final double width;
}

// this is for illustration. the code is lazy.
int get flex => <FormFactor, int> {
      FormFactor.small: 2,
      FormFactor.medium: 3,
      FormFactor.large: 4,
    } [FormFactor.from(MediaQuery.of(context).size)];
```